docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -161,3 +190,10 @@ To get the name of the GPU, type::
 For example, you should see output like the line below::
```

Comment:
If the installation is successful, then you should see the following output::

Reply:
Hey @marianalevineu , just added 'for example' here. Once that's included then we should be all set and I can merge if you are unable to.

Reply:
@manasvita just modified it, let me know if that's not what you had in mind.

Reply:
Yes, it's fine now. 
Also, it would be good to add "You should see the result True if CUDA is detected a GPU